spec/core/1.0.md

```diff
@@ -455,9 +452,29 @@ The server is under no obligation of agreeing to any module or capability.
 Clients should be prepared to gracefully handle unavailable modules or capabilities.
 If the client finds that the server has not agreed to a module or capability that the client needs, it must exit.
 
-## 5. Message types for `vt6/core1.0`
+## 5. Other eternal message types
+
+### 5.1. The `nope` message
+
+- Directionality: any
+- Required capabilities: none
+- Number of arguments: none
+
+Upon receiving an invalid message, a server or client MAY reply by sending a `nope` message with no arguments.
+It MUST do so if:
+
+1. the recipient cannot determine the message type of the invalid message, or
```

Comment:
"the recipient" being "It" being "a server or client". Please use only one of the forms for clarity, preferrably "recipient", that is 463s/server or client/recipient/ 466,467s/the recipient/it/